### ISSUE TITLE
Not getting error when select different file.

### ISSUE THREAD

**Issue by altaf933** [ArthurHub/Android-Image-Cropper#200]
When i am going to select different file but not an images it not return any error. Please consider this issue as enhancement.

We can set onActivityResult  as Cancel or Error method so it will finish our current activity.

**Comment by ArthurHub** [comment]
I don't understand what you mean, you can't select non-image file...

**Comment by altaf933** [comment]
I mean non images file if i do open it give a message like "Image load successful" but actually it is a another extension file so it should be an Error message. 
Checkout below reproducible step that may help .

[GIF](https://giphy.com/gifs/26gJAG3gy9f6QFeOQ)

**Comment by ArthurHub** [comment]
I'm curious, how common is this scenario? How it can happen in normal flow without using super user apps like file explorer?

**Comment by altaf933** [comment]
My personal device Moto 2Generation have  Camera,Gallery and Photo default android application if I  will remove super users apps.
Also it should a more option depend on the installed application.

**Comment by ArthurHub** [comment]
Sure, but camera, photos and gallery apps allow only valid image files.
You can't select an apk to load...

**Comment by altaf933** [comment]
Yes of course but user can select file by super user app (i.e File explorer) . 
Can we set any error message if file is not an images?

If Yes:
  We can fix this issue in BitmapCroppingWorkerTask.java file at onPostExecute by  check result.bitmap==null that can set message like "Invalid images or choose image file only".

**Comment by RaviLalwani556** [comment]
Any fix on the same ?

**Comment by asos-davidf** [comment]
Any fix underway? It's quite likely in a normal flow: Google Drive for instance can't restrict types of files shown, it will display everything.

**Comment by thawatchai** [comment]
Super user apps like File Explorer are built-in in most phones nowadays. Could you consider fixing this issue? Thank you very much.
